File: core/src/main/java/ir/logicfan/core/ui/common/BroadCastReceiverRegisterListener.java
package ir.logicfan.core.ui.common;

import android.content.BroadcastReceiver;
import android.content.IntentFilter;

public interface BroadCastReceiverRegisterListener {

    void registerReceiver(BroadcastReceiver receiver, IntentFilter filter);

    void unregisterReceiver(BroadcastReceiver receiver);
}

File: core/src/main/java/ir/logicfan/core/ui/common/BroadCastReceiverRegisterListener.kt
package ir.logicfan.core.ui.common

import android.content.BroadcastReceiver
import android.content.IntentFilter

interface BroadCastReceiverRegisterListener {

    fun registerReceiver(receiver: BroadcastReceiver, filter: IntentFilter)

    fun unregisterReceiver(receiver: BroadcastReceiver)
}
